### ISSUE TITLE
Unhandled Exception error after downloading episodes (gtk3 branch)

### ISSUE THREAD

**Issue by dannn-o** [gpodder/gpodder#274]
Running gPodder (ver. 3.9.3) as a git clone with the gtk3 branch (Linux Mint 17.3), the following error dialog appears once the download of one or more episodes is complete:
```
Unhandled exception
Please report this problem and restart gPodder:
boxed cannot be created directly; try using a constructor, see: help(Gtk.TreeRowReference)
```

![gpodder error](https://cloud.githubusercontent.com/assets/17674969/25110723/2aa3ab84-23b4-11e7-989c-5d322fe05e83.png)
After clicking OK to the error, the entries on the "Progress" tab remain, and the title bar still shows "downloading 1 file" with the most recent percent complete and transfer rates showing.
The corresponding error log (2017-04-17.txt) is attached.
[2017-04-17.txt](https://github.com/gpodder/gpodder/files/927065/2017-04-17.txt)

Exiting and restarting gPodder clears out the "Progress" tab, and the title bar appears normal again.

**Comment by vossad01** [comment]
Thank you for this!  This does not reproduce for me (I presume because I am no a newer version of GTK3); however, I believe I have fixed this.

You can test by updating your repository (`git pull origin gtk3`) then launching gPodder.

**Comment by dannn-o** [comment]
Good news:  The fix appears to have worked.

The sync problem I mentioned in #265 is still occurring, though.  As you suggested, I will pursue that with a new, separate issue.  Thanks for the quick turnaround.

**Comment by vossad01** [comment]
Great!  Thanks for testing this!
